README.md

```diff
@@ -225,19 +226,24 @@ cd ${crate}
 pytest --profile=devel rust-vmm-ci/integration_tests/test_coverage.py
 ```
 
-If the PR coverage is higher than the upstream coverage, the coverage file
-needs to be manually added to the commit before submitting the PR:
+Sample coverage_config files:
+
+- [x86](coverage_config_x86_64.json.sample)
+- [aarch64](coverage_config_aarch64.json.sample)
 
+If the PR coverage is higher than the upstream coverage, the coverage file
+needs to be manually added to the commit before submitting the PR. For example:
 ```bash
-git add tests/coverage
+git add coverage_config_x86_64.json
 ```
 
 Failing to do so will generate a fail on the CI pipeline when publishing the
 PR.
 
 **NOTE:** The coverage file is only updated in the `devel` test profile. In
 the `ci` profile the coverage test will fail if the current coverage is higher
-than the coverage reported in [tests/coverage](tests/coverage).
+than the coverage reported in [coverage_config_x86_64.json](coverage_config_x86_64.json)
+_(or coverage_config_aarch64.json)_.
```

Comment:
I would think it is clearer to link to both files.
```suggestion
than the coverage reported in [`coverage_config_x86_64.json`](coverage_config_x86_64.json) or 
[`coverage_config_aarch64.json`](coverage_config_aarch64.json) respectively.
```

Reply:
The PR initially had aarch64 linked as well, but I asked @nathanwhyte to remove it because that's not actually used anymore, so it would be rather confusing. When we will add support again for aarch64 we can add the comment as well. For now I would rather have it as is.